HelpSource/Classes/Platform.schelp

```diff
@@ -178,6 +137,16 @@ code::
 thisProcess.platform.killProcessByID(~pid);
 ::
 
+method:: recompile
+recompile the class library
+
+method:: clearMetadata
+clear metadata at the given path
+
+method:: devLoc
+???
```

Comment:
there is usage for this, it's related to SVN which we no longer use. We should deprecate it.